ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -327,6 +327,29 @@ EOF
   echo "Enabled AWS Spot instances for worker nodes"
 }
 
+# enable_efa_pg_instance_config is an AWS specific option that enables one worker machineset in a placement group and with EFA Network Interface Type, other worker machinesets will be ENA Network Interface Type by default.....
+function enable_efa_pg_instance_config() {
+  local dir=${1}
+  #sed -i 's/          instanceType: .*/          networkInterfaceType: EFA\n          placementGroupName: pgcluster\n          instanceType: c5n.9xlarge/' "$dir/openshift/99_openshift-cluster-api_worker-machineset-0.yaml"
+  pip3 install pyyaml --user
+  pushd "${dir}/openshift"
+  python -c '
+import os
+import yaml
+
+for manifest_name in os.listdir("./"):
+    if "machineset" in manifest_name:
+      data = yaml.safe_load(open(manifest_name))
+      data["spec"]["template"]["spec"]["providerSpec"]["value"]["networkInterfaceType"] = "EFA"
+      data["spec"]["template"]["spec"]["providerSpec"]["value"]["instanceType"] = "c5n.9xlarge"
+      data["spec"]["template"]["spec"]["providerSpec"]["value"]["placementGroupName"] = "pgcluster"
+      open(manifest_name, "w").write(yaml.dump(data, default_flow_style=False))
+      break
+' || exit 2
+    popd
+  echo "Patched efa pg into $dir/openshift/99_openshift-cluster-api_worker-machineset-0.yaml"
```

Comment:
Shall we use `manifest_name` to replace `99_openshift-cluster-api_worker-machineset-0.yaml` hard code here?

Reply:
ok, I am trying if `pip3` works, if it works, will update this.

Reply:
It works! Let me update it.
```
Collecting pyyaml
  Downloading https://files.pythonhosted.org/packages/cd/e5/af35f7ea75cf72f2cd079c95ee16797de7cd71f29ea7c68ae5ce7be1eda0/PyYAML-6.0.1.tar.gz (125kB)
Installing collected packages: pyyaml
  Running setup.py install for pyyaml: started
    Running setup.py install for pyyaml: finished with status 'done'
Successfully installed pyyaml-6.0.1
~/installer/openshift /output
/output
Patched efa pg into /tmp/installer/openshift/99_openshift-cluster-api_worker-machineset-0.yaml
```